motr/client_internal.h

```diff
@@ -229,6 +229,9 @@ struct m0_op_idx {
 	struct m0_sm_group  *oi_sm_grp;
 	struct m0_ast_rc     oi_ar;
 
+	/* Minimum successful CAS requests. */
```

Comment:
Suggestion:
```
/**
  * Minimum number of successful CAS operations to treat
  * parent DIX operation as successful.
  * /
```
Also please consider _oi_min_success_ if everybody is ok with it.